package.json

```diff
@@ -4,9 +4,13 @@
   "description": "React components for govuk layouts and elements",
   "main": "index.js",
   "scripts": {
-    "test": "eslint .",
+    "lint": "eslint .",
+    "test": "jest --config ./scripts/jest/config.source.js --env=jsdom --watch",
+    "test:update": "npm run test -- --updateSnapshot",
```

Comment:
What's this for?

Reply:
*tl;dr*
With snapshot testing the first time it executes it generates a snapshot of how the html should look. You can manually change the snapshot when you make a change that results in different html (that's perfect for small changes - i.e. most of the time). But sometimes its just easier to blat the original snapshot, generate a new one and validate the diff. That script generates the new one.

More detail: 
https://facebook.github.io/jest/docs/en/snapshot-testing.html